src/Extensions/ServiceFabricGuest.Abstractions/factory/ServiceFabricClientFactory.cs

```diff
@@ -1,23 +1,20 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT license.
-
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
 using Microsoft.Extensions.Options;
 
 namespace Microsoft.Omex.Extensions.ServiceFabricGuest.Abstractions
-{
+{
 	/// <summary>
 	/// Factory for service fabric client wrapper
 	/// </summary>
 	public class ServiceFabricClientFactory
-	{
+	{

```

Comment:
why are there two changes here? Can you rollback them?

Reply:
Also I think that they are after code format